lib/src/metta/runner/stdlib/stdlib.metta

```diff
@@ -667,6 +667,26 @@
    (let $c1 (collapse $arg1) (let $c2 (collapse $arg2)
      (let $u (subtraction-atom $c1 $c2) (superpose $u)))))
 
+(@doc add-reducts
+  (@desc "Function takes space and expression which contains key-value pairs, reduces them and adds result into given space")
```

Comment:
Expression should not necessarily contain key-value pairs. In case of this function, these can be arbitrary atoms

Reply:
@Necr0x0Der  Okay, so just `Function takes space and expression, evaluates atoms in it and adds them into given space` then?